ui-tests/src/test/java/io/syndesis/qe/steps/other/ODataSteps.java

```diff
@@ -0,0 +1,76 @@
+package io.syndesis.qe.steps.other;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.cucumber.datatable.DataTable;
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.accounts.AccountsDirectory;
+import io.syndesis.qe.utils.HTTPResponse;
+import io.syndesis.qe.utils.HttpUtils;
+import io.syndesis.qe.utils.ODataUtils;
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.Headers;
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class ODataSteps {
+
+    @When("^create OData credentials$")
+    public void createODataCredentials() {
+        Account oData = new Account();
+        oData.setService("OData");
+        Map<String, String> properties = new HashMap<>();
+        properties.put("serviceUri", ODataUtils.getOpenshiftService());
+        oData.setProperties(properties);
+        AccountsDirectory.getInstance().addAccount("odata", oData);
+        log.info("Created new Account: odata");
+    }
+
+    @When("^.*insert entity \"([^\"]*)\" into \"([^\"]*)\" collection on OData service$")
+    public void insertEntityToODataService(final String entity, final String collection) {
+        String requestBody = ODataUtils.readResourceFile(this.getClass().getClassLoader().getResource(entity));
+        Headers headers = Headers.of("Content-Type", "application/json");
+        HTTPResponse response = HttpUtils.doPostRequest(ODataUtils.getOpenshiftRoute() + collection, requestBody, headers);
+        assertThat(response.getCode()).isEqualTo(201);
+        log.info("Entity from resource file " + entity + " succesfully inserted to sample OData service");
+    }
+
+    @Then("^.*check that \"([^\"]*)\" entity in \"([^\"]*)\" collection contains$")
```

Comment:
This might be useful for check of update operation, but since I am not planning to implement it right away and its a simple method, I guess it can be removed for now.